factcast-site/documentation/content/setup/properties.md

```diff
@@ -0,0 +1,78 @@
++++
+draft = false
+title = "Properties"
+description = ""
+date = "2017-04-24T18:36:24+02:00"
+weight = 210
+
+creatordisplayname = "Uwe Schaefer"
+creatoremail = "uwe@codesmell.de"
+
+[menu.main]
+parent = "usage"
+identifier = "properties"
+
++++
+
+
+Properties you can use to configure Factcast:
+
+### Schema/Transformation-Registry
+
+| Property-Name        | Semantics           | Default   
+| ------------- |:-------------|:-----|
+| factcast.store.pgsql.schemaRegistryUrl      | if a schemaRegistryUrl is defined, FactCast goes into validating mode. The only protocols allowed here are *'http', 'https' and 'classpath'*|  
+| factcast.store.pgsql.persistentRegistry      | if fetched Schema and Transformation Documents are persisted into Postgres | false 
+| factcast.store.pgsql.persistentTransformationCache      | if Transformed Fact payloads are persistently cached into Postgres| false 
+| factcast.store.pgsql.allowUnvalidatedPublish      | If validation is enabled, this controls if publishing facts, that are **not validatable** (due to missing meta-data or due to missing schema in the registry) are allowed to be published or should be rejected.  |  false 
+| factcast.store.pgsql.persistentSchemaStore |   If validation is enabled, this controls if a local snapshot of the schemaregistry is persisted to psql or just kept in mem.   |    true 
```

Comment:
Is this the same as `persistentRegistry` or am I missing something?
Apart from that maybe its worth it to group the properties by context. For example `persistentTransformationCache` should go down to `deleteTransformationsStaleForDays` and friends?

Reply:
ummm... will fix that